.github/workflows/pr-title-update.yml

```diff
@@ -0,0 +1,25 @@
+name: Update PR Title If Target Branch Is A Version Branch
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  update-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update PR title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          TARGET_BRANCH=$(gh pr view $PR_NUMBER --json baseRefName -q .baseRefName)
+          PR_TITLE=$(gh pr view $PR_NUMBER --json title -q .title)
+          
+          if [[ "$TARGET_BRANCH" != "dev" ]]; then
```

Comment:
I think it would be better if you use https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-branches to check that the target branch starts with `release_` .